syft/pkg/cataloger/generic/cataloger.go

```diff
@@ -122,17 +123,9 @@ func (c *Cataloger) Catalog(ctx context.Context, resolver file.Resolver) ([]pkg.
 
 		log.WithFields("path", location.RealPath).Trace("parsing file contents")
```

Comment:
optional opportunity for unrelated cleanup -- this should be using `logger` not `log`